scripts/check_license.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+# exit immediately when a command fails
+set -e
+# only exit with zero if all commands of the pipeline exit successfully
+set -o pipefail
+# error on unset variables
+set -u
+
+licRes=$(
```

Comment:
I think the correct way of doing it is to use one of the existing go tools instead of rolling our own custom checks. Ideally it should read from a `license header` file and match header 1:1. Right now looks like we are looking at only a subset of a pattern.

Reply:
You are right. We should be checking the entire license header text. I've created the following issue to deal with it in the future: https://github.com/pravega/zookeeper-operator/issues/29

Reply:
This comment is twofold. First part is using a standard go tool and second is checking the whole license file. The issue just covers the second part.

Reply:
@arvindkandhare what standard go tool are you referring to?

Reply:
I agree on both counts (checking entire license and using a tool for it) but I also don't think this should stop this being merged and handled separately and more extensively in #29